views/index.py

```diff
@@ -442,6 +440,7 @@ def remove_copy(item_id, copy_id):
 
 
 @library.route('/wishlist', methods=['GET', 'POST'])
+@require_logged_in()
 def wishlist():
     try:
```

Comment:
Those `try/except` clauses all over the source code could be now redundant, since the decorators for roles are implementend?
I'm not sure of this, though, needs to be checked.